### ISSUE TITLE
HGV restriction axleload not taken into account

### ISSUE THREAD

**Issue by aoles** [GIScience/openrouteservice#262]
It seems that the hgv restriction `axleload` is not functional at the moment. Consider driving on [Kleinschmidtstraße](https://www.openstreetmap.org/way/28171940) tagged as `maxaxleload=13` with `axleload` set to a value higher than 13t: https://maps.openrouteservice.org/directions?n1=49.405853&n2=8.684945&n3=17&a=49.406017,8.68613,49.404411,8.687804&b=4a&c=0&f4=100&k1=en-US&k2=km

**Comment by TimMcCauley** [comment]
@aoles do please make sure this is also not working with the API directly, the webapp might have a bug.

**Comment by aoles** [comment]
@TimMcCauley I already did, the parameter passed in the URL reads `options={"profile_params":{"restrictions":{"axleload":"100"}},"vehicle_type":"hgv"}`.

**Comment by aoles** [comment]
The fix includes changing the condition in [`VehicleParameters`](https://github.com/GIScience/openrouteservice/blob/development/openrouteservice/src/main/java/heigit/ors/routing/parameters/VehicleParameters.java#L890) from 
```java
return _height >0.0 || _length > 0.0 || _width > 0.0 || _weight >0.0 || _characteristics != 0;
```
to
```java
return _height >0.0 || _length > 0.0 || _width > 0.0 || _weight >0.0 || _axleload >0.0 || _characteristics != 0;
```

**Comment by aoles** [comment]
~It seems that the update in `VehicleParameters` mentioned above is not enough, investigating further...~

**UPDATE:** The fix is ok, though `maxaxleload` values > 12.8 are not resolved properly because of #263.
